bower.json

```diff
@@ -6,11 +6,12 @@
     "ember-cli-test-loader": "0.2.2",
     "ember-mocha": "~0.8.11",
     "lodash": "^4.10.0",
-    "perfect-scrollbar": "~0.6.7",
+    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```

Comment:
We are getting this because we are < 1.0.0. We fixed this in 1.0.0 by removing this dependency.